src/main/java/org/devcourse/resumeme/business/user/controller/mentee/dto/MenteeInfoUpdateRequest.java

```diff
@@ -4,4 +4,16 @@
 
 public record MenteeInfoUpdateRequest(String nickname, String phoneNumber, Set<String> interestedPositions, Set<String> interestedFields, String introduce) {
 
+    public MenteeInfoUpdateRequest(String nickname, String phoneNumber, Set<String> interestedPositions, Set<String> interestedFields, String introduce) {
+        this.nickname = nickname;
+        this.phoneNumber = phoneNumber;
+        this.interestedPositions = replaceNullWithEmptySet(interestedPositions);
+        this.interestedFields = replaceNullWithEmptySet(interestedFields);
+        this.introduce = introduce;
+    }
+
+    private Set<String> replaceNullWithEmptySet(Set<String> setAttributes) {
+        return setAttributes == null ? Set.of() : setAttributes;
```

Comment:
```suggestion
        if (attributes== null) {
           return Set.of();
       }
       
       return attributes;
```
set, list와 같이 타입은 안적는게 좋아보여요

Reply:
[chore: 파라미터명 및 엔드포인트 수정 요청 반영](https://github.com/resumeme/Backend/pull/175/commits/d53caa72269b1b5adfa1ee864387c760b5be4e5a)